introduction.md

```diff
@@ -1,3 +1,12 @@
 Hi, my name is Earl Lewis.
 My favorite Code School path is Javascript.
 I'm learning how to use git via CodeSchool.
+
+My deadly skills are:
+* HTML
+* Javascript
+* CSS
+* SQL
+* PL/SQL
+* MongoDB
+* Node.js
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?